applications/MeshMovingApplication/tests/test_ale_fluid_solver/ProjectParameters.json

```diff
@@ -0,0 +1,183 @@
+{
+    "problem_data"     : {
+        "problem_name"  : "ale_fluid_test",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 0,
+        "start_time"    : 0.0,
+        "end_time"      : 1
+    },
+    "output_processes" : { },
+    "solver_settings"  : {
+        "model_part_name"             : "FluidModelPart",
+        "domain_size"                 : 2,
+        "solver_type"                 : "Monolithic",
+        "ale_settings"                : {
+            "solver_type"         : "mesh_solver_structural_similarity",
+            "ale_interface_parts" : [
+                [
+                    "AutomaticInlet2D_inlet",
```

Comment:
Why a list of lists ? can't all the names (of sub model parts ) be in one list ?

Reply:
As you pointed out later, this is to impose ale-BCs by components